docs/decisions/0009-only-one-anonymous-user-id-per-user-and-courseid.rst

```diff
@@ -0,0 +1,24 @@
+One Anonymous User Id per user, course_id pair
```

Comment:
I added a decision directory to the student djangoapp.  I wonder if this decision should also live in that app?

Reply:
Yep, its should.

Reply:
Oh, I didn't see you were already working on this. I will remove this from this PR